pkgs/top-level/all-packages.nix

```diff
@@ -1437,7 +1437,7 @@ in
 
   elm-github-install = callPackage ../tools/package-management/elm-github-install { };
 
-  emby = callPackage ../servers/emby { };
+  emby = throw "The Emby derivation has been removed, see jellyfin instead for a free software fork.";
```

Comment:
This should probably be in `aliases.nix`.

Reply:
The concern I have with using an alias instead of throw is that the two are not compatible. We don't want to potentially mangle our users data without them realizing it.

Reply:
Oh no, we can put `throw` in `aliases.nix`.

Reply:
@worldofpeace Good point, moved the throw to aliases.

Reply:
@worldofpeace sorry I thought you meant to *not* use a throw, but just alias `emby` as `jellyfin`. In which case ignore my comment :smile: